test/net-test.js

```diff
@@ -283,6 +283,113 @@ describe('Net', function() {
       check(pkt);
     });
 
+    it('getCFilters', () => {
+      const check = (pkt) => {
+        assert.equal(pkt.cmd, 'getcfilters');
+        assert.equal(pkt.type, packets.types.GETCFILTERS);
+
+        assert.equal(pkt.startHeight, 12);
+        assert.equal(pkt.stopHash.toString('hex'), 'ab'.repeat(32));
+        assert.equal(pkt.filterType, common.FILTERS.BASIC);
+
+        assert.equal(pkt.getSize(), 37);
+        assert.bufferEqual(pkt.toRaw(), Buffer.from('000c000000abababababababababababababababababababababababababababababababab', 'hex'));
```

Comment:
hehe i dunno can we break this up?

```suggestion
        assert.bufferEqual(
          pkt.toRaw(),
          Buffer.from('000c000000ababababababababababababab' +  
                      'ababababababababababababababababababab', 'hex')
          );
```